core/src/main/java/org/mule/internal/connection/ConnectionManagerAdapter.java

```diff
@@ -19,4 +20,6 @@
 {
 
     RetryPolicyTemplate getDefaultRetryPolicyTemplate();
+
+    PoolingProfile getDefaultPoolingProfile();
```

Comment:
i know it's out of you issue's scope, but since you're at it please add javadoc for the other method above too